src/upload/methods/bundlr.rs

```diff
@@ -197,7 +197,20 @@ impl BundlrMethod {
             .as_str()
             .expect("Failed to get an id from bundlr transaction.");
 
-        let link = format!("https://arweave.net/{}", id);
+        // Get extension for the asset type.
+        let ext = asset_info
+            .content_type
+            .split('/')
+            .nth(1)
+            .ok_or_else(|| anyhow!("Failed context type to get extension"))?;
+
+        let link = match asset_info.data_type {
+            DataType::Image => format!("https://arweave.net/{id}?ext={ext}"),
```

Comment:
Would it make sense to simplify the code of the match? E.g.:
```
let link = match asset_info.data_type {
    DataType::Image | DataType::Animation => {
        format!("https://arweave.net/{id}?ext={ext}")
     },
    DataType::Metadata => format!("https://arweave.net/{}", id),
}
```

Reply:
yeah, totally, will fix